src/IO/Config.hs

```diff
@@ -39,7 +40,19 @@ defaultConfig = Config {
 }
 
 getDir :: IO FilePath
-getDir = (++ "/.taskell") <$> getHomeDirectory
+getDir =
```

Comment:
Hi! Someone posted this on reddit asking for a code review, so here I am!

I'd put this entire function into an IO block: 

```haskell
getDir = do
```